pom.xml

```diff
@@ -47,9 +47,9 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
     <!-- versions -->
-    <version.io.opentracing-opentracing>0.31.0</version.io.opentracing-opentracing>
-    <version.io.opentracing.contrib-opentracing-concurrent>0.2.0</version.io.opentracing.contrib-opentracing-concurrent>
-    <version.io.opentracing.contrib-opentracing-web-servlet-filter>0.2.0</version.io.opentracing.contrib-opentracing-web-servlet-filter>
+    <version.io.opentracing-opentracing>0.32.0</version.io.opentracing-opentracing>
+    <version.io.opentracing.contrib-opentracing-concurrent>0.3.0</version.io.opentracing.contrib-opentracing-concurrent>
+    <version.io.opentracing.contrib-opentracing-web-servlet-filter>0.3.0</version.io.opentracing.contrib-opentracing-web-servlet-filter>
     <version.io.microprofile-opentracing-api>1.3</version.io.microprofile-opentracing-api>
```

Comment:
MP-OT API does not depend directly on the OT API. Here we use only traced annotation.